src/field.rs

```diff
@@ -7,6 +7,8 @@
 //! [`FftFriendlyFieldElement`], and have an associated element called the "generator" that
 //! generates a multiplicative subgroup of order `2^n` for some `n`.
 
+#![allow(deprecated)] // for Field96
```

Comment:
What does the compiler say if we remove this?

Reply:
Each use of Field96 or its field in the macro expansion flags a warning. Those can be addressed with method-level `#[allow(deprecated)]` attributes. The one I couldn't figure out how to silence without a module-level attribute was the `Field96` argument to the macro invocation itself. (The macro invocation does not accept an `#[allow]` attribute on it, and anything inside the parentheses becomes a macro argument)

Reply:
I'm a little nervous about silencing deprecation warnings module-wide. If we move the declaration of `Field96` into a sub-module of `field.rs`, can we re-export `Field96` with `pub use crate::field::sub_module::Field96`, and then tag just the sub-module and the `pub use` as `#![allow(deprecated]`?